doc/internal/man3/ossl_algorithm_do_all.pod

```diff
@@ -41,11 +41,11 @@ The function described here are mainly useful for discovery, and
 possibly display of what has been discovered, for example an
 application that wants to display the loaded providers and what they
 may offer, but also for constructors, such as
-L<ossl_construct_method(3)>.
+L<ossl_method_construct(3)>.
 
 =head1 SEE ALSO
 
-L<ossl_construct_method(3)>, L<EVP_MAC_do_all(3)>
+L<ossl_method_construct(3)>, L<EVP_MAC_do_all_provided(3)>
```

Comment:
@levitte all these EVP_MAC functions which were introduced in e74bd29053a don't seem to exist anymore?. Should they be removed entirely, since they never entered a stable branch?

https://github.com/openssl/openssl/blob/b62dcede015b0718e944df1c1d91b1e46eeb7deb/util/libcrypto.num#L4436-L4444

Reply:
@mspncp : sure, feel free to submit a PR for their removal.

Reply:
Ok. What about the numbers? Shall I renumber or leave the gap as it is?

Reply:
You can start with refreshing your master checkout :wink:

Reply:
> You can start with refreshing your master checkout

Don't worry. My four worktrees are _always_ up-to-date:

**./update-repo**
```bash
#!/bin/bash

set -o errexit
set -o xtrace

git fetch --all

git checkout master
git pull --ff-only

(cd ../openssl-1.1.1 && git checkout OpenSSL_1_1_1-stable && git pull --ff-only)
(cd ../openssl-1.1.0 && git checkout OpenSSL_1_1_0-stable && git pull --ff-only)
(cd ../openssl-1.0.2 && git checkout OpenSSL_1_0_2-stable && git pull --ff-only)
```

But when I review a pull request, I tend to check it out locally (b62dcede015b) in order to get the full picture.

_(wiseguy mode off... :wink:  )_

But thanks for the hint!

Reply:
Done, see pull request #10421.

Reply:
So I assume that by now, you have noticed that those numbers aren't numbers any more...

Reply:
Yes. I did follow the discussion in the pr, but I wasn't aware that your change was already merged.

Reply:
This might not be the right place for this.  A flag to the script that says "complain if any placeholder function does not exist when assigning numbers" might be useful.